ansible-ipi-install/roles/installer/tasks/02_get_oc.yml

```diff
@@ -0,0 +1,27 @@
+---
+- name: Get the ocp client tar gunzip file
+  get_url:
+    url: "https://mirror.openshift.com/pub/openshift-v4/clients/ocp-dev-preview/{{ version }}/openshift-client-linux-{{ version }}.tar.gz"
```

Comment:
for a verson, i'd like to specifi the latest-4.3 (or latest-4.4)  which works for the subdirectory but not the tar.
I think get the release name from the release.txt file on the Name: line

(could make this an issue /enhancement)

Reply:
Can you add more details here on how to do that? Yes, I agree this could be an enhancement.

Reply:
[sreichar@sreichar-t460p ~] $ curl -s https://mirror.openshift.com/pub/openshift-v4/clients/ocp-dev-preview/latest-4.3/release.txt | awk '/Name:/ {print $NF}'
4.3.0-0.nightly-2020-01-21-205041
[sreichar@sreichar-t460p ~] $